### DISCUSSION TITLE
Coravel Cron Not running at specific time

### DISCUSSION THREAD

**Discussion by jsebastianmr** [jamesmh/coravel#287] [Q&A] [answered]
Hi,

I'm using coravel as schedule worker, I have a problen when I try to use a cron expresion to trigger the task, for example if I put my cron expression like "* * * * *" the task runs every minute, but if I want to lunch it daily at specific time it doesnt work.

For example
"* * * * *" Works every minute.
"30 20 * * *"  It should lunch the task at 20:30 daily but it doesnt work!

Thank you very much if you can help me with this issue

The next code is the implementation:
```
            providerCoravel.UseScheduler(scheduler =>
            {
                scheduler
                .Schedule<Notification>()
                .Cron("30 20 * * *");
            });
```

**Comment by jamesmh** [comment, marked as answer]
That looks correct... why don't you try to use the global error handling: https://docs.coravel.net/Scheduler/#global-error-handling

I think the following would work:

```
            providerCoravel.UseScheduler(scheduler =>
            {
                scheduler
                .Schedule<Notification>()
                .Cron("30 20 * * *");
            })
            .OnError(exception => throw exception);
```

Or just write the exception to the console. Normally errors are ignored to fail gracefully, but here with the `OnError` method you can be more specific about what happens when an exception occurs.

You might find that there's a missing DI registration or something.

Let me know if that helps to find what's going on.

**Reply by jsebastianmr** [reply]
Thanks for the answer, I put the OnError method but It did not throw anything.

I did a test putting a malformed cron expression and it threw a malformed expression excepción, but with the cron at specific time like "30 20 * * *" nothin happens, no excepción or no execution :(

**Reply by jamesmh** [reply]
Ok, just to make sure - the time you put into Cron is UTC right? If you are expecting the time you put into Cron to be within a certain time zone then you need to use -> https://docs.coravel.net/Scheduler/#zoned-schedules

**Reply by jsebastianmr** [reply]
Thanks a lot!,  You are right and the link helped me , this worked for me:

```
            providerCoravel.UseScheduler(scheduler =>
            {
                scheduler
                .Schedule<Notification>()
                .Cron("06 22 * * *")     // I'll get the cron expression from appsettings file
                .Zoned(TimeZoneInfo.Local);
            }).OnError(exception => throw exception);
```

**Reply by jamesmh** [reply]
Great!
